src/bp/cluster.ts

```diff
@@ -41,7 +41,7 @@ export const setupMasterNode = (logger: sdk.Logger) => {
   process.SERVER_ID = process.env.SERVER_ID || nanoid('1234567890abcdefghijklmnopqrstuvwxyz', 10)
 
   // Fix an issue with pkg when passing custom options for v8
-  cluster.setupMaster({ execArgv: [] })
+  cluster.setupMaster({ execArgv: process.execArgv.filter(x => !x.startsWith('--max_old_space_size')) })
```

Comment:
I think it's a good idea to remove all execArgs but only in production / packaged version

Reply:
I have no preference, so i'll clear arguments when it's packaged